Makefile

```diff
@@ -407,9 +410,9 @@ DATAPLANE_KUTTL_CONF                             ?= ${OPERATOR_BASE_DIR}/datapla
 DATAPLANE_KUTTL_DIR                              ?= ${OPERATOR_BASE_DIR}/dataplane-operator/tests/kuttl/tests
 DATAPLANE_KUTTL_NAMESPACE                        ?= dataplane-kuttl-tests
 BM_CTLPLANE_INTERFACE                            ?= enp1s0
-BM_ROOT_PASSWORD_SECRET                          ?=
+BM_ROOT_PASSWORD                                 ?=
 GENERATE_SSH_KEYS				 ?= true
-DATAPLANE_EXTRA_NOVA_CONFIG_FILE                 ?= /dev/null
+DATAPLANE_EXTRA_NOVA_CONFIG_FILE                 ?=
```

Comment:
while it is a problem if we create an empty extra config by default?

Reply:
actually nova extra configmap is not creating when passing file. so only  creating configmap when file name is provided

Reply:
> while it is a problem if we create an empty extra config by default?

changed to original.